repose-aggregator/components/filters/body-extractor-to-header-filter/src/test/scala/org/openrepose/filters/bodyextractortoheader/BodyExtractorToHeaderSchemaTest.scala

```diff
@@ -20,17 +20,20 @@
 package org.openrepose.filters.bodyextractortoheader
 
 import org.junit.runner.RunWith
+import org.mockito.Mockito._
 import org.openrepose.commons.test.ConfigValidator
+import org.openrepose.core.services.config.ConfigurationService
 import org.scalatest.junit.JUnitRunner
 import org.scalatest.{FunSpec, Matchers}
 
 @RunWith(classOf[JUnitRunner])
 class BodyExtractorToHeaderSchemaTest extends FunSpec with Matchers {
-  val validator = ConfigValidator("/META-INF/schema/config/body-extractor-to-header.xsd")
+  val filter: BodyExtractorToHeaderFilter = new BodyExtractorToHeaderFilter(mock(classOf[ConfigurationService]))
```

Comment:
Right, so it would be a bit redundant, but I would suggest doing something like this:
```
class MyFilter extends AbstractConfiguredFilter {
  override def SCHEMA_LOCATION = MyFilter.SchemaLocation
  override def DEFAULT_CONFIG = MyFilter.DefaultConfig
}

object MyFilter {
  final val SchemaLocation = "/path/to/schema.xsd"
  final val DefaultConfig = "/path/to/default/config.cfg.xml"
}
```

Then in the test, you can use the static definitions. Ideally, you would only have static definitions (at least, at public visibility) but I don't know how you would get that to work.